packages/office-ui-fabric-react/src/components/Button/ButtonDefault.tsx

```diff
@@ -0,0 +1,29 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+import {
+  css,
+  getNativeProps,
+  buttonProperties
+} from '../../Utilities';
+
+import { ButtonBase } from './ButtonBase';
+import './sass/ButtonBase.scss';
+import './sass/ButtonDefault.scss';
+
+export class DefaultButton extends ButtonBase {
+  protected getRootClassName() { return 'ms-Button--default'; }
+
+  public render() {
```

Comment:
How should we support icons and button descriptions. Should those be on the ButtonBase, and triggered when the prop is passed in (which is troubling because the icon/description styles are in the prospective button scss) or should buttons needing more than just "child" to be rendered have their own custom render that passes in the extra markup?